src/ApiCore/ApiStatus.php

```diff
@@ -114,4 +114,16 @@ public static function statusFromRpcCode($code)
         }
         return ApiStatus::UNRECOGNIZED_STATUS;
     }
+
+    /**
+     * @param string $status
+     * @return int
+     */
+    public static function rpcCodeFromStatus($status)
+    {
+        if (array_key_exists($status, self::$apiStatusToCodeMap)) {
+            return self::$apiStatusToCodeMap[$status];
+        }
+        return Code::UNKNOWN;
```

Comment:
I am not sure that this should return UNKNOWN. That is a specific error code that corresponds to a HTTP 500 response (see https://github.com/googleapis/googleapis/blob/master/google/rpc/code.proto#L51). I would be tempted to either return `-1` or throw an exception.

Reply:
I like `-1` for parity with `statusFromRpcCode`

Reply:
All set: https://github.com/googleapis/gax-php/pull/118/commits/366ca4b610bdd7e0a128941b7426814243db1916